torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -1026,6 +1026,8 @@ def should_use_remote_autotune_cache(inductor_meta):
         return inductor_meta.get("autotune_remote_cache")
     if not inductor_meta.get("is_fbcode"):
         return False
+    if torch._utils_internal.is_fb_unit_test():
+        return False
```

Comment:
I'm concerned that we have to keep remembering each of these locations to update, because I /know/ we're going to be adding more cache knob functions as well

Reply:
after the runtime refactor, not being able to read config options directly, sharing code has gotten tricker. Any recommendations? Do you want me to move this somewhere lower?

Reply:
I don't quite understand the runtime refactor constraints. Is there an explanation of them somewhere / can you explain?

Reply:
@jansel did a refactor so that runtime files do not depend on compile time files, so we cannot directly read configs, not sure whether @jansel wrote anything down

Reply:
> @jansel did a refactor

I believe that was a pre-requisite so that we could arrange things such that the workers responsible for doing Triton parallel compile wouldn't need to import all of torch, which can take ~3s, at least on my devserver. jansel had a candidate change that spawned a new process for each triton-compile job, so this overhead would be important to minimize. We didn't end up landing that diff, so I'd guess it's conceivable to undo this refactor if we want. Though I do think it would still be beneficial to optimize the import overhead. Currently I think it's not _critical_ because we start the subprocess pool early and pay the overhead once, but it would probably still help for short-running scripts.